espresso/streamer.go

```diff
@@ -331,7 +362,62 @@ func (s *BatchStreamer[B]) processHotShotRange(ctx context.Context, start, finis
 			continue
 		}
 
-		s.processEspressoTransactions(ctx, height, txns)
+		for _, txn := range txns.Transactions {
+			s.processEspressoTransaction(ctx, txn)
+		}
+	}
+
+	return nil
+}
+
+// processHotShotRange is a helper method that will load all of the blocks from
```

Comment:
Comment indicates that the method name is `processHotShotRange`, but the actual method name is `streamHotShotRange`.